### PR TITLE
fix failing image e2e test

### PR DESCRIPTION
**Pull request by vrothberg**
The timestamps of some images must have changed changing the number of
expected filtered images.  The test conditions seem fragile but for now
it's more important to get CI back.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

**Comment by vrothberg** [comment]
I was now struck by the e2e test for multiple network aliases.  It should now be fixed with the second commit.

File: test/e2e/images_test.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ WORKDIR /test
 		result := podmanTest.Podman([]string{"image", "list", "-q", "-f", "after=quay.io/libpod/alpine:latest"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(result.OutputToStringArray()).Should(HaveLen(7), "list filter output: %q", result.OutputToString())
```
Comment (TomSweeneyRedHat) [inline test/e2e/images_test.go:207]: I am always leery about bumping expected results for a test that used to work.  Are we sure that there's not a problem elsewhere?

Comment (vrothberg) [reply test/e2e/images_test.go:207]: Yes, @baude rebuild images over the weekend since Quay was complaining.

**Comment by Luap99** [comment]
The network alias test is still failing on ubuntu

**Comment by vrothberg** [comment]
Closing in favor of https://github.com/containers/podman/pull/9385. Thanks @baude !

**Pull request closed without merging**